alembic/versions/b022867d09e3_changed_library_file_to_reference_in_.py

```diff
@@ -0,0 +1,28 @@
+"""changed library_file to reference in part_info
+
+Revision ID: b022867d09e3
+Revises: e83aa47e530b
+Create Date: 2019-11-25 20:34:47.384076+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'b022867d09e3'
+down_revision = 'e83aa47e530b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
please remove these comments about auto generation. They aren't true, you haven't kept any of the autogenerated code.